src/klicky.ts

```diff
@@ -1,3 +1,54 @@
 export const Klicky = () => {
   console.log("klickly ran!");
 };
+
+const outer = document.querySelector('.outer');
+
+const capture = {
+  capture : true
+};
+
+const noneCapture = {
+  capture : false 
+};
+
+const once = {
+  once : false 
+};
+
+outer?.addEventListener('click', onceHandler, once);
+
+function onceHandler(event)
+{
+  console.log("You have clicked the page");
+  write();
+  var date = new Date();
```

Comment:
Another bit of ES5 vs ES6, we tend to avoid using `var` nowadays as it is the weakest way to declare a variable. Like you were doing above, can you try using `const` or `let` from now on?

Reply:
Yup! I've changed variable declarations to "let"